### PR TITLE
Arrange styles of header component for mobile and of welcome page

### LINKED ISSUE #42: Arrange styling of the welcome page (mobile version)

### LINKED ISSUE #47: Create mobile version of the header component for the 3 pages

### PR DESCRIPTION
**Pull request by JimeBlue**
## Description

This PR arranges the styles of the header component so that it is suitable for the mobile version and also arranges the styles of the welcome page.

### Styles for the header component

- I added the classes `w-1/2 ` `lg:w-1/4` and `2xl:w-full`  to the header´s title in the welcome page in order to manage the width of the title while adopting a mobile fist approach.
- I changed `-bottom-32` to `-bottom-38` in the image of the welcome page so that it is aligned in all screen versions.
- I added the classes `pl-2 ` and `sm:pl-0` to the `titleStyles` in the **list** and **add-item** pages to detach the title from the left border in small and medium mobile screens.

### Styles for the welcome page
- Removed the `label `in `ShareTokenForm `to match figma design.
- Added `flex ` `flex-col` and `text-center` to `section` to center elements in it and added padding with `pt-16` .
- I changed the placeholder of the `input `in `ShareTokenForm `to match the figma design.
- Added margin to the `input` in `ShareTokenForm` with the classes `mb-4` and `sm:mb-0`.
                                          

## Related Issue

Close #42 
Close #47

## Acceptance Criteria

- The header should be suitable for mobile screens.
- The welcome page should resemble the figma design. 

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    ✓| :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

header-mobile-before

header-non-home-page-before

welcome-page-before

### After

header-home-after
header-non-home-after

welcome-page-after

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with git pull origin` jm-style-header-for-mobile-and-style-welcome-page` and check that branch out with git checkout ` jm-style-header-for-mobile-and-style-welcome-page`.
- Then npm start to launch the app.
- Use google chrome dev tool to check responsiveness.

### CODE REVIEW

File: src/components/ShareTokenForm.js [hunk 1]
```diff
@@ -36,12 +36,11 @@ const ShareTokenForm = () => {
...
-      <label htmlFor="shareToken">Share token</label>
       <input
-        className="input"
+        className="input mb-4 sm:mb-0"
         type="text"
         id="shareToken"
-        placeholder="three word token"
+        placeholder="Enter 3 word token"
         onChange={(e) => setSelectedToken(e.target.value)}
         value={selectedToken}
```
Comment (redapy) [inline src/components/ShareTokenForm.js]: I guess it's good to have the label anyway for a11y reason, you can wrap the input with a label (without any text).

**Review by redapy — CHANGES_REQUESTED** [review]
All looks great Jimena, great job! I had only one small note.

File: src/components/Home.js [hunk 2]
```diff
@@ -33,14 +33,16 @@ const Home = () => {
...
+      <section className="flex flex-col text-center pt-16">
         <Button
           createToken={createToken}
           text="Create new List"
           width={'w-52'}
         />
-        <p>- or -</p>
-        <p>Join an existing shopping list by entering a three word token.</p>
+        <p className="pt-8 font-bold">- or -</p>
+        <p className="pt-6 pb-6 font-bold w-64">
```
Comment (hellodeborahuk) [inline src/components/Home.js]: Could use `py-6` here so slim down the code.

File: src/components/ShareTokenForm.js (hunk 1 above)
Comment (hellodeborahuk) [reply src/components/ShareTokenForm.js]: I agree with Reda. Could we use an aria label instead or hide the label?

**Review by hellodeborahuk — COMMENTED** [review]
Great work Jimena!

**Pull request merged**